hw/lint/doc/README.md

```diff
@@ -1,18 +1,93 @@
 # RTL Linting
 
-Linting is a productivity tool for designers to quickly find typos and
-bugs at the time when the RTL is written. Running lint is important
-when using SystemVerilog, a weakly-typed language, unlike other hardware
-description languages. We consider linting to be critical for conformance
-to our goals of high quality designs.
+Linting is a productivity tool for designers to quickly find typos and bugs at the time when the RTL is written.
+Running lint is important when using SystemVerilog, a weakly-typed language, unlike other hardware description languages.
+We consider linting to be critical for conformance to our goals of high quality designs.
 
-Currently, due to the proprietary nature of tool collateral, all linting
-activity is done offline and reported back to designers.  The project will
-standardize on a particular linting tool, and results will be shared in
-some form through continuous integration build results, published tool
-outputs, pre-submit checks, and/or linting summaries of tool output
-(TODO: publication details).  At that time this README will be updated
-with setup and run instructions.
+We have standardized on the AscentLint tool from RealIntent for this task due to its fast run-times and comprehensive set of rules that provide concise error and warning messages.
+
+The lint flow run scripts and waiver files are available in the GitHub repository of this project, but due to the proprietary nature of the lint rules and their configuration, the _"Low Risc Lint Rules"_ lint policy file can not be publicly provided.
+However, the _"Low Risc Lint Rules"_ are available as part of the default policies in AscentLint release 2019.A.p3 or newer (as `LRLR-v1.0.policy`).
+This enables designers that have access to this tool to run the lint flow provided locally on their premises.
+
+In order to run lint on a comportable IP block, the corresponding Fusesoc core file must have a lint target and include (optional) waiver files as shown in the following example taken from the Fusesoc core of the AES comportable IP:
```

Comment:
people new to the project landing on this page may not know what `fusesoc` is. please
provide a pointer to that tool or something in our collateral that explains our usage of it.

Reply:
done.